.github/workflows/debian-executable-release.yml

```diff
@@ -0,0 +1,47 @@
+name: Debian Executable Release
```

Comment:
renamed to platform-executable - >debian-executable-release since we are only handling debian release here, we have separate workflow for other (macos and windows).